edgelet/kube-client/src/client.rs

```diff
@@ -77,6 +78,58 @@ where
     S::Error: Fail,
     Body: From<<S as Service>::ResBody>,
 {
+    pub fn is_subject_allowed(
+        &mut self,
+        resource: Option<String>,
```

Comment:
Why options for function args? Doest it make sense to call this method if either of args is `None`?